hw/ip/entropy_src/dv/entropy_src_sim_cfg.hjson

```diff
@@ -73,6 +73,13 @@
       uvm_test_seq: entropy_src_rng_vseq
     }
 
+    {
+      name: entropy_src_rng_with_xht_rsps
+      uvm_test: entropy_src_rng_test
+      uvm_test_seq: entropy_src_rng_vseq
+      run_opts: ["+xht_only_default_rsp=0"]
+    }
+
```

Comment:
I think we should associate this test with a V3 testpoint that validates XHT, but the testplan does not contain such a testpoint yet. Thoughts?

As is, the test is unmapped

Reply:
In this case, I would add a new testpoint. E.g. something like this:
```
    {
      name: external_health_checks
      desc: '''
            Verify continuous and windowed external health checks.
            See also https://github.com/lowRISC/opentitan/issues/16276
            '''
      stage: V3
      tests: ["entropy_src_rng_with_xht_rsps"]
    }
```

Because:
- I would like at least a couple of seeds in the nightlies to run with the extht interface enabled. Just to avoid that we completely break this when marching towards M2.5.
- According to #16276 , this is still a V3 item. If in the future we ever decide that we don't want this for a V3, we can decide what do with the testpoint then. But until then we should keep testing this.

Reply:
Thanks for your suggestions; I've added the testpoint and mapped `entropy_src_rng_with_xht_rsps` to it